client/assets/javascripts/utils/CommentsManager.js

```diff
@@ -10,10 +22,7 @@ const CommentsManager = {
    * @returns {Promise} - jqXHR result of ajax call.
    */
   fetchComments() {
-    return Promise.resolve($.ajax({
-      url: API_URL,
-      dataType: 'json',
-    }));
+    return fetch(API_URL).then(checkStatus).then(parseJSON);
```

Comment:
I see how others use a global polyfill for fetch, but I think this is really awkward.

I'd rather see something like what lodash has.

The way that webpack has to use both the import and export loaders is a real brain twister. I get it and am posting to the forum on how this works.

However, this sort of goes against my philosophy of "Don't make me think". 

Looking for more discussion on this one. @alexfedoseev ? @skv-headless ?